README.md

```diff
@@ -1 +1,11 @@
-# examples
\ No newline at end of file
+# Useful Roc Examples
+
+This repository stores a number of useful Roc examples to complement the [Roc Tutorial](https://www.roc-lang.org/tutorial) and other guides. 
```

Comment:
I've given this a lot of thought and I think it would be good to have two folders:
- `roc_concepts`: where every example is minimal and focused on a single roc concept like serialization, unit tests, doc comments, opaque types... . I wouldn't demonstrate multiple concepts in one example because I believe more information at once makes it more difficult to learn something.
- `how_to_x_in_roc` (WIP name): for general purpose oriented things: json serialization, minimal web server, minimal native GUI, but also [rosetta code type examples](https://rosettacode.org/wiki/Rosetta_Code) like fibonacci, calculating pi, binary search...

Perhaps we should even split the folders over two repos. Let me know what you think @lukewilliamboswell